crates/db_schema/replaceable_schema/triggers.sql

```diff
@@ -889,3 +889,126 @@ CALL r.create_inbox_combined_trigger ('person_post_mention');
 
 CALL r.create_inbox_combined_trigger ('private_message');
 
+-- search: (post, comment, community, person)
+CREATE PROCEDURE r.create_search_combined_trigger (table_name text)
+LANGUAGE plpgsql
+AS $a$
+BEGIN
+    EXECUTE replace($b$ CREATE FUNCTION r.search_combined_thing_insert ( )
+            RETURNS TRIGGER
+            LANGUAGE plpgsql
+            AS $$
+            BEGIN
+                -- TODO need to figure out how to do the other columns here
+                INSERT INTO search_combined (published, thing_id)
+                    VALUES (NEW.published, NEW.id);
+                RETURN NEW;
+            END $$;
+    CREATE TRIGGER search_combined
+        AFTER INSERT ON thing
+        FOR EACH ROW
+        EXECUTE FUNCTION r.search_combined_thing_insert ( );
+        $b$,
+        'thing',
+        table_name);
+END;
+$a$;
+
+CALL r.create_search_combined_trigger ('post');
+
+CALL r.create_search_combined_trigger ('comment');
+
+CALL r.create_search_combined_trigger ('community');
+
+CALL r.create_search_combined_trigger ('person');
+
+-- You also need to triggers to update the `score` column.
+-- post | post_aggregates::score
+-- comment | comment_aggregates::score
+-- community | community_aggregates::users_active_monthly
+-- person | person_aggregates::post_score
+--
+-- Post score
+CREATE FUNCTION r.search_combined_post_score_update ()
+    RETURNS TRIGGER
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+    UPDATE
+        search_combined
+    SET
+        score = NEW.score
+    WHERE
+        post_id = NEW.post_id;
+    RETURN NULL;
+END
+$$;
+
+CREATE TRIGGER search_combined_post_score
+    AFTER UPDATE OF score ON post_aggregates
+    FOR EACH ROW
+    EXECUTE FUNCTION r.search_combined_post_score_update ();
```

Comment:
@dullbananas For now, I just did a trigger update on the specific columns of the `_aggregates` table, to update the `search_combined.score` field. LMK if these are okay.